frontend/src/services/loginAPI.tsx

```diff
@@ -0,0 +1,27 @@
+export const API_BASE_URL = "http://localhost:3000";
+
+interface LoginResponse {
+  accessToken: string;
+}
+
+export const login = async (username: string, password: string): Promise<LoginResponse> => {
+  try {
+    const response = await fetch(`${API_BASE_URL}/user/login`, {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify({ username, password }),
+    });
+
+    if (response.status === 400) {
+      throw new Error("Wrong username or password");
```

Comment:
Nel form attualmente viene mostrato Error: Wrong.... quindi mettere la lettera minuscola qua ha più senso.
```suggestion
      throw new Error("wrong username or password");
```